databricks_cli/dbfs/api.py

```diff
@@ -75,12 +88,40 @@ class DbfsErrorCodes(object):
     RESOURCE_DOES_NOT_EXIST = 'RESOURCE_DOES_NOT_EXIST'
     RESOURCE_ALREADY_EXISTS = 'RESOURCE_ALREADY_EXISTS'
     PARTIAL_DELETE = 'PARTIAL_DELETE'
+    TOO_MANY_REQUESTS = 'TOO_MANY_REQUESTS'
+
+
+def my_before_sleep(retry_state):
```

Comment:
I think we can rename this method to something specific to our use case. E.g. `before_sleep_429`